randovania/generator/pickup_pool/pickup_creator.py

```diff
@@ -137,17 +137,19 @@ def create_nothing_pickup(resource_database: ResourceDatabase, model_name: str =
     )
 
 
-def create_visual_etm() -> PickupEntry:
+def create_visual_etm(game: RandovaniaGame = RandovaniaGame.METROID_PRIME_ECHOES,
```

Comment:
Something against renaming the function to a name which makes sense for none prime players?